horizomer/utils/tree.py

```diff
@@ -91,6 +144,66 @@ def intersect_trees(tree1, tree2):
     return (tree1_lap, tree2_lap)
 
 
+def collapse_short_branches(tree, cutoff):
+    """Collapse internal nodes with branch length below cutoff.
+
+    Parameters
+    ----------
+    tree : skbio.TreeNode
+        tree to search for nodes to collapse
+    cutoff : int or float
+        branch length cutoff under which node should be collapsed
```

Comment:
node_s_! plural, right?

Reply:
I think it should be singular in this context? It refers to _a_ node that needs to be collapse because its branch length is below cutoff.